### PR TITLE
Simplify Track Cost calcs

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Multiply out 5 in track cost calcs and use _GBP (as _GBP is *10 this ends up dividing all constants by 2)
This should make things much easier to follow in the calcs.
I bumped up the type to 64bit perhaps an overkill but couldn't be bothered to work out if it would cause truncation if this was not changed.

@X123M3-256 take note this will affect your coaster PR if merged.

### CODE REVIEW

File: src/openrct2/ride/coaster/meta/Hypercoaster.h [hunk 1]
```diff
@@ -39,7 +39,7 @@ constexpr const RideTypeDescriptor HypercoasterRTD =
     SET_FIELD(RatingsCalculationFunction, ride_ratings_calculate_corkscrew_roller_coaster),
     SET_FIELD(RatingsMultipliers, { 50, 30, 10 }),
     SET_FIELD(UpkeepCosts, { 40, 20, 80, 11, 3, 10 }),
-    SET_FIELD(BuildCosts, { 105, 5, 50, }),
+    SET_FIELD(BuildCosts, { 62.5_GBP, 2.5_GBP, 50, }),
```
Comment (Fred-1044276) [inline src/openrct2/ride/coaster/meta/Hypercoaster.h:42]: Shouldn't it be 52.5_GBP?

Comment (tupaschoal) [reply src/openrct2/ride/coaster/meta/Hypercoaster.h:42]: Definitely looks like a typo
```suggestion
    SET_FIELD(BuildCosts, { 52.5_GBP, 2.5_GBP, 50, }),
```

**Review by tupaschoal — COMMENTED** [review]
This is excellent, it will make things so much easier to understand. If I'm to be super picky, most of the `_GBP` literals in the repo are using two decimals (I found just one that isn't). I'll push the fix and you can drop it if you'd like

**Comment by duncanspumpkin** [comment]
@tupaschoal thanks for that.

**Review by tupaschoal — APPROVED** [review]

**Comment by Broxzier** [comment]
This looks so much better. 🎉

**Pull request merged**